scripts/mockgen.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+mockgencmd="mockgen"
+
+${mockgencmd} -source=x/challenge/types/expected_keepers.go -destination=x/challenge/types/expected_keepers_mocks.go -package=types
+${mockgencmd} -source=x/payment/types/expected_keepers.go -destination=x/payment/types/expected_keepers_mocks.go -package=types
+${mockgencmd} -source=x/permission/types/expected_keepers.go -destination=x/permission/types/expected_keepers_mocks.go -package=types
+${mockgencmd} -source=x/sp/types/expected_keepers.go -destination=x/sp/types/expected_keepers_mocks.go -package=types
+${mockgencmd} -source=x/storage/types/expected_keepers.go -destination=x/storage/types/expected_keepers_mocks.go -package=types
```

Comment:
add virtualgroup module here

Reply:
mock tests for virtualgroup  module are not currently available, add them as needed

Reply:
added